urlparse4/cgurl.pyx

```diff
@@ -14,6 +15,8 @@ from six.moves.urllib.parse import urljoin as stdlib_urljoin
 from six.moves.urllib.parse import urlunsplit as stdlib_urlunsplit
 from six.moves.urllib.parse import urlparse as stdlib_urlparse
 from six.moves.urllib.parse import urlunparse as stdlib_urlunparse
+# gotta do the condition on python 3 because these cant be imported with six
+from urllib.parse import _coerce_args, unquote_to_bytes
 from w3lib.util import to_bytes, to_native_str, to_unicode
```

Comment:
Just noticed this import, not related to this PR but it's fine to copy the source of these functions, instead of making w3lib a dependency (if we don't need it elsewhere)